tests/functional/collectBucketMetricsAndUpdateBucketCapacityInfo.js

```diff
@@ -94,8 +94,8 @@ describe('collectBucketMetricsAndUpdateBucketCapacityInfo', () => {
                 next();
             }),
             next => client.getBucketAttributes(testBucketName, logger, (err, bucketInfo) => {
-                assert.equal(err, null);
-                assert.equal(bucketInfo.getCapabilities(), null);
+                assert.equal(err && err.message, 'InternalError');
+                assert.equal(bucketInfo, undefined);
                 next();
```

Comment:
seems weird to expect an `InternalError` on `getBucketAttributes()`...

Reply:
I updated the test, it was not the right check

Reply:
Interesting point though, it was passing for the wrong reasons: the assert in the callback, when failing, would never reach the `done`, but instead, throw n error that will be catched by whatever can do it; and here, it was in the mongoclientinterface clas... so we end up calling this test callback a second time, and with the right error.

Note that this is not the first time i see this and we have this problem in almost all our node codebase.